graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -114,7 +114,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name of the GraphiQL version.
          */
-        public static final String VERSION = PREFIX + ".version";
+        public static final String VERSION_CONFIG = PREFIX + ".version";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.